cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -1077,13 +1081,14 @@ func (uc *UpstreamController) queryNode() {
 	}
 }
 
-func (uc *UpstreamController) unmarshalPodStatusMessage(msg model.Message) (ns string, podStatuses []edgeapi.PodStatusRequest) {
+func (uc *UpstreamController) unmarshalPodStatusMessage(msg model.Message) (ns, nodeID string, podStatuses []edgeapi.PodStatusRequest) {
```

Comment:
Could rewrite this func with none-named value return?

Reply:
Sorry, I don't get it, isn't it more clarify and clean up use named return parameters? Could you explain why you think should be rewriting it please.